src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp

```diff
@@ -272,6 +346,27 @@ class NonlinearSolverParameters : public dataRepository::Group
 
   /// Flag to specify whether subcycling is allowed or not in sequential schemes
   integer m_subcyclingOption;
+
```

Comment:
I personally prefer long list of parameters to be grouped in a struct

Reply:
I agree with @paveltomin on this.

Reply:
i moved those params into CompositionalMultiphaseFVM and grouped them into a struct